.github/workflows/release.yml

```diff
@@ -36,16 +36,23 @@ jobs:
           fetch-depth: 0
       - uses: actions/setup-go@v4
         with:
-          go-version: "1.21"
-      - name: Build & annotate
+          go-version: "1.21.1"
+      - name: Install wasm-opt
+        run: |
+          wget https://github.com/WebAssembly/binaryen/releases/download/version_116/binaryen-version_116-x86_64-linux.tar.gz
+          tar -xzf binaryen-version_116-x86_64-linux.tar.gz
+          sudo mv binaryen-version_116/bin/wasm-opt /usr/local/bin
```

Comment:
nitpicking, can't we install it via system packages? Ubuntu [has the package](https://packages.ubuntu.com/search?suite=kinetic&section=all&arch=any&keywords=binaryen&searchon=names) available

Reply:
@flavio I tried but it installs an old version (105) which has some memory leak and the runner gets reaped because of an OOM

Reply:
See: https://github.com/kubewarden/kyverno-dsl-policy/actions/runs/6472248845

Reply:
I see, can you leave a comment about that?